server/WebAPI/appsettings.json

```diff
@@ -20,6 +20,11 @@
     "Password": "mail_password"
   },
   "Front": {
-    "AddressFront": "http://localhost:8080"
+    "AddressFront": "http://me.test.com"
+  },
+  "GoogleOauth": {
+    "ClientId": "333247262677-ri736kk9bh4c129e44j7i6d18idt4ptu.apps.googleusercontent.com",
+    "ClientSecret": "secret",
```

Comment:
you can use ENV variables instead of hardcoding sensitive secret data. i'd suggest use ENV for ClientId as well